common/lib/util/eventemitter.ts

```diff
@@ -176,19 +176,21 @@ var EventEmitter = (function() {
 	 * @param event the name of the event to listen to
 	 * @param listener the listener to be called
 	 */
-	EventEmitter.prototype.once = function(event, listener) {
-		var argCount = arguments.length, self = this;
+	once(event: string, listener: Function) {
+		const argCount = arguments.length;
```

Comment:
I would prefer `argumentCount`. 😄

Reply:
I've refactored this to use `args` in 32f8d04 so I'd prefer to keep this one as `argCount`.